.github/workflows/news-check-pr.yml

```diff
@@ -9,19 +9,40 @@ jobs:
     runs-on: ubuntu-latest
     name: Run Test and Static Analysis
     steps:
-      - uses: actions/checkout@v3     
-      - uses: subosito/flutter-action@v2
+      - uses: actions/checkout@v3
+
+      - id: 'auth'
```

Comment:
This authenticates with firebase, I had to create a secret with the `gcp_credentials` json file content, which is something you download from the `gcp console`